pkg/crypto/signer.go

```diff
@@ -82,3 +91,37 @@ func (d *defaultSigner) Sign(data []byte) (signature []byte, err error) {
 	signature[64] = v
 	return signature, nil
 }
+
+// SignTx signs an ethereum transaction
+func (d *defaultSigner) SignTx(transaction *types.Transaction) (*types.Transaction, error) {
```

Comment:
It would be good to have tests for defultSigner's SignTx and EthereumAddress, maybe just to have the points what should be done in next iterations?

Reply:
I added tests for both